openmetadata-ui/src/main/resources/ui/src/components/Applications/AppDetails/AppDetails.component.tsx

```diff
@@ -348,7 +348,6 @@ const AppDetails = () => {
             {appData && (
               <AppSchedule
                 appData={appData}
-                onCancel={onBrowseAppsClick}
```

Comment:
Don't we need onCancel anymore?

Reply:
@chirag-madlani  we are not using onCancel is AppSchedule anymore, so we don't require to pass it here.